Bank/src/main/java/com/ark/bank/BankController.java

```diff
@@ -463,4 +463,18 @@ public void update(Observable o, Object arg) {
             notifyObservers(arg);
         }
     }
+
+    @Override
+    public boolean changeCreditLimit(String sessionKey, BankAccount bankAccount, long limit) {
+        if (!isSessionActive(sessionKey)
+                || (bankAccount == null)
+                || !sessionKeyMatchesBankAccountNumber(sessionKey, bankAccount.getNumber())
+                || limit < 0
+                || limit > 100000) {
```

Comment:
Waarom dit maximum?

Reply:
En als er al een maximum is, maak er dan een static final long van ergens... nu leest het niet goed. (magic numbers....)